lib/modules/manager/kustomize/artifacts.ts

```diff
@@ -0,0 +1,217 @@
+import is from '@sindresorhus/is';
+import { quote } from 'shlex';
+import upath from 'upath';
+import { TEMPORARY_ERROR } from '../../../constants/error-messages';
+import { logger } from '../../../logger';
+import { exec } from '../../../util/exec';
+import type {
+  ExecOptions,
+  ExtraEnv,
+  ToolConstraint,
+} from '../../../util/exec/types';
+import {
+  deleteLocalFile,
+  getSiblingFileName,
+  localPathExists,
+  privateCacheDir,
+  readLocalFile,
+  writeLocalFile,
+} from '../../../util/fs';
+import { getRepoStatus } from '../../../util/git';
+import type { UpdateArtifact, UpdateArtifactsResult, Upgrade } from '../types';
+import { parseKustomize } from './extract';
+
+function generateHelmEnvs(): ExtraEnv {
+  return {
+    HELM_EXPERIMENTAL_OCI: '1',
```

Comment:
do we really need this? should only be set if user sets contraints which causes a helm version <3.8.0

Reply:
Let me know if you think i should remove it, otherwise i'm under the impression that it should be there as-is.

Reply:
yes, only set if `config.constraints?.helm` is set and `semver.satisfies(config.constraints.helm, '<3.8.0')`